lib/interface/cli/completion/tree.json

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
this should not be committed and should be inside .gitignore.
no point in commits this file